requestlog/requestlog.go

```diff
@@ -107,6 +112,7 @@ type Entry struct {
 	ResponseHeaderSize int64
 	ResponseBodySize   int64
 	Latency            time.Duration
+	TraceID            string
```

Comment:
Since this is our abstract representation of a request log entry, we are permitted to have more strong types here. Let's use OpenCensus's `TraceID` type directly to make it clear what format it is in.

```suggestion
	TraceID            trace.TraceID
```